src/Mutant/MutantCreator.php

```diff
@@ -93,4 +78,28 @@ private function isCoveredByTest(Mutation $mutation, CodeCoverageData $codeCover
 
         return $codeCoverageData->hasTestsOnLine($filePath, $line);
     }
+
+    private function getMutatedCode(Mutation $mutation, string $mutatedFilePath): string
```

Comment:
I would suggest to name it `createMutatedCode` as it more suite its content (file is created, content is generated)